simple_https_server/src/lib.rs

```diff
@@ -0,0 +1,180 @@
+#![allow(non_upper_case_globals)]
+
+// Import hacspec and all needed definitions.
+use std::{io::prelude::*, net::TcpListener, str, time::Duration};
+
+use base::{AppError, RecordStream};
+use bertie::{tls13api::*, tls13utils::*};
+#[cfg(feature = "evercrypt")]
+use evercrypt_cryptolib::*;
+#[cfg(not(feature = "evercrypt"))]
+use hacspec_cryptolib::*;
+use hacspec_lib::*;
+use rand::*;
+
+const SHA256_Aes128Gcm_EcdsaSecp256r1Sha256_X25519: Algorithms = Algorithms(
+    HashAlgorithm::SHA256,
+    AeadAlgorithm::Aes128Gcm,
+    SignatureScheme::EcdsaSecp256r1Sha256,
+    NamedGroup::X25519,
+    false,
+    false,
+);
+
+/*
+const SHA256_Chacha20Poly1305_RsaPssRsaSha256_X25519: Algorithms = Algorithms(
+    HashAlgorithm::SHA256,
+    AeadAlgorithm::Chacha20Poly1305,
+    SignatureScheme::RsaPssRsaSha256,
+    NamedGroup::X25519,
+    false,
+    false,
+);
+*/
+
+const default_algs: Algorithms = SHA256_Aes128Gcm_EcdsaSecp256r1Sha256_X25519;
+
+// Cert and Key
+const ECDSA_P256_SHA256_CERT: [u8; 522] = [
```

Comment:
These blobs should really go into files. But we can do this in a followup.

Reply:
https://github.com/cryspen/bertie/issues/50